app/src/main/java/com/andrew/tasky/presentation/adapters/AttendeeItemAdapter.kt

```diff
@@ -25,26 +25,25 @@ class AttendeeItemAdapter(
 
     override fun onBindViewHolder(holder: AttendeeItemViewHolder, position: Int) {
         holder.binding.apply {
-
-            //Adds and removes deleteAttendeeButtonView, and creatorTV to appropriate attendee
             if(attendees[position].attendeeType == AttendeeType.CREATOR) {
                 deleteAttendeeButton.isVisible = false
                 creatorTextView.isVisible = true
             }
-            else{
+            else if (!isAttendee){
                 deleteAttendeeButton.isVisible = true
                 creatorTextView.isVisible = false
             }
+            else{
+                deleteAttendeeButton.isVisible = false
+                creatorTextView.isVisible = false
+            }
```

Comment:
Every time, you have an if/else where you just set booleans to true/false, that's very suspicious and you can get rid of the if/else:
```
val isCreator = attendees[position].attendeeType == AttendeeType.CREATOR
deleteAttendeeButton.isVisible = !isCreator
creatorTextView.isVisible = isCreator
```